token_server/README.md

```diff
@@ -1,4 +1,32 @@
 # Simple web server to provide EarthEngine tokens to non-whitelisted users
 
-Run with `node --experimental_modules token_server/token_server.js`.
+## Deploy to Google App Engine
 
+* Follow initial steps for Google App Engine deployment: install `gcloud`, log
+in as the GD user `gd-earthengine-user@givedirectly.org`, and set the gcloud
```

Comment:
Can you specify where/what to log in to and where/how to set the gcloud project?

Reply:
I prefer not to duplicate widely available documentation, especially given Google's propensity to deprecate and replace. But added currently valid instructions and link to general SDK docs.

Reply:
It's not about being able to read documentation, more about finding the correct documentation and set up for our specific project. If there's a page that documents the steps you have here then we should just link that instead and hopefully that will get updated. I think it's nice to save users the steps of digging through different Google App Engine documentation pages if we can.

Reply:
I don't think there is anything more specific than what I linked to, at least not that I could find when looking again now. Would you like more detail than what is in the latest version of this file?

Reply:
I thought when you said you didn't want to duplicate documentation there was documentation saying the same steps as you'd written here, in which case we should just link to it but if there's not then nice to leave the steps IMO